src/Symfony/Bundle/FrameworkBundle/Resources/config/console.php

```diff
@@ -0,0 +1,290 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Bundle\FrameworkBundle\Command\AboutCommand;
+use Symfony\Bundle\FrameworkBundle\Command\AssetsInstallCommand;
+use Symfony\Bundle\FrameworkBundle\Command\CacheClearCommand;
+use Symfony\Bundle\FrameworkBundle\Command\CachePoolClearCommand;
+use Symfony\Bundle\FrameworkBundle\Command\CachePoolDeleteCommand;
+use Symfony\Bundle\FrameworkBundle\Command\CachePoolListCommand;
+use Symfony\Bundle\FrameworkBundle\Command\CachePoolPruneCommand;
+use Symfony\Bundle\FrameworkBundle\Command\CacheWarmupCommand;
+use Symfony\Bundle\FrameworkBundle\Command\ConfigDebugCommand;
+use Symfony\Bundle\FrameworkBundle\Command\ConfigDumpReferenceCommand;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerDebugCommand;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerLintCommand;
+use Symfony\Bundle\FrameworkBundle\Command\DebugAutowiringCommand;
+use Symfony\Bundle\FrameworkBundle\Command\EventDispatcherDebugCommand;
+use Symfony\Bundle\FrameworkBundle\Command\RouterDebugCommand;
+use Symfony\Bundle\FrameworkBundle\Command\RouterMatchCommand;
+use Symfony\Bundle\FrameworkBundle\Command\SecretsDecryptToLocalCommand;
+use Symfony\Bundle\FrameworkBundle\Command\SecretsEncryptFromLocalCommand;
+use Symfony\Bundle\FrameworkBundle\Command\SecretsGenerateKeysCommand;
+use Symfony\Bundle\FrameworkBundle\Command\SecretsListCommand;
+use Symfony\Bundle\FrameworkBundle\Command\SecretsRemoveCommand;
+use Symfony\Bundle\FrameworkBundle\Command\SecretsSetCommand;
+use Symfony\Bundle\FrameworkBundle\Command\TranslationDebugCommand;
+use Symfony\Bundle\FrameworkBundle\Command\TranslationUpdateCommand;
+use Symfony\Bundle\FrameworkBundle\Command\WorkflowDumpCommand;
+use Symfony\Bundle\FrameworkBundle\Command\YamlLintCommand;
+use Symfony\Bundle\FrameworkBundle\EventListener\SuggestMissingPackageSubscriber;
+use Symfony\Component\Console\EventListener\ErrorListener;
+use Symfony\Component\Messenger\Command\ConsumeMessagesCommand;
+use Symfony\Component\Messenger\Command\DebugCommand;
+use Symfony\Component\Messenger\Command\FailedMessagesRemoveCommand;
+use Symfony\Component\Messenger\Command\FailedMessagesRetryCommand;
+use Symfony\Component\Messenger\Command\FailedMessagesShowCommand;
+use Symfony\Component\Messenger\Command\SetupTransportsCommand;
+use Symfony\Component\Messenger\Command\StopWorkersCommand;
+use Symfony\Component\Translation\Command\XliffLintCommand;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('console.error_listener', ErrorListener::class)
+        ->args([
```

Comment:
We need an extra indent for all methods called after each `->set()`, check an example of the desired formatting here: https://github.com/symfony/symfony/pull/36778/files

Reply:
Updated